pkgs/applications/misc/lttoolbox/default.nix

```diff
@@ -0,0 +1,46 @@
+{ fetchFromGitHub, lib, pkgs, stdenv }:
+
+stdenv.mkDerivation rec {
+  pname = "lttoolbox";
+  version = "3.7.1";
+
+  src = fetchFromGitHub {
+    owner = "apertium";
+    repo = "lttoolbox";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-3lHXKtwQSrMGQEGOGr27e3kB2qKkTFZcEzeAnIm89Rg=";
+  };
+  name = "lttoolbox";
+  nativeBuildInputs = [
+    pkgs.autoconf
+    pkgs.automake
+    pkgs.pkg-config
+    pkgs.utf8cpp
+    pkgs.libtool
+  ];
+  buildInputs = [
+    pkgs.libxml2
+    pkgs.libxml2.dev
+    pkgs.icu
+    pkgs.utf8cpp
```

Comment:
I suppose this was moved to `nativeBuildInputs`?

Reply:
Ah yes. Updated.